docs/prow/presets.md

```diff
@@ -25,3 +25,4 @@ This document contains the list of all Presets available in the [`config.yaml`](
 | **preset-bot-github-ssh** | It connects the ssh key of the bot account to your job and sets the value with the path to this key.                    |
 | **preset-kyma-artifacts-bucket** | It defines the environment variable for the Kyma's artifact bucket.                     |
 | **preset-stability-checker-slack-notifications** | It defines a webhook URL and a client token required for the Slack integration.                 |
+| **preset-slack-internal-bot-token** | It sets the environment variable with the Slack token for the bot account in internal workspace. More information can bo found [here](https://api.slack.com/docs/token-types#bot).|
```

Comment:
preset-slack-token-for-prowjob-failures-notification ?

Reply:
discussed